### PR TITLE
Add auto bake error handling to asset server

### PR DESCRIPTION
**Pull request by huffman**

### CODE REVIEW

File: assignment-client/src/assets/AssetServer.cpp [hunk 1]
```diff
@@ -140,7 +142,7 @@ BakingStatus AssetServer::getAssetStatus(const AssetPath& path, const AssetHash&
     auto bakedPath = "/.baked/" + hash + "/" + bakedFilename;
     auto jt = _fileMappings.find(bakedPath);
     if (jt != _fileMappings.end()) {
-        if (jt->toString() == hash) {
+        if (jt->first == hash) {
```
Comment (Atlante45) [inline assignment-client/src/assets/AssetServer.cpp]: This should be `second`

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2939/HighFidelity-Beta-PR11281-2aa39e7da488dae25f90baac8f2b2e64b9efc0c3.exe

File: assignment-client/src/assets/AssetServer.cpp (hunk 1 above)
Comment (huffman) [reply assignment-client/src/assets/AssetServer.cpp]: good catch

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2973/HighFidelity-Beta-PR11281-0cd8a5cd413dde40383ff87edac35e97e0adaa61.exe

File: assignment-client/src/assets/AssetServer.cpp [hunk 2]
```diff
@@ -516,14 +529,14 @@ void AssetServer::handleGetMappingOperation(ReceivedMessage& message, SharedNode
 void AssetServer::handleGetAllMappingOperation(ReceivedMessage& message, SharedNodePointer senderNode, NLPacketList& replyPacket) {
     replyPacket.writePrimitive(AssetServerError::NoError);
 
-    auto count = _fileMappings.size();
+    uint32_t count = _fileMappings.size();
```
Comment (Atlante45) [inline assignment-client/src/assets/AssetServer.cpp]: This adds a warning

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2986/HighFidelity-Beta-PR11281-d2c7235fed3d1346693cc38c2f4431488bc4fd93.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2991/HighFidelity-Beta-PR11281-25f64e1aa9b12d4f595bd2916e16a6d9fdc73407.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2993/HighFidelity-Beta-PR11281-bf53161a0b21dee4d0da0081967f669b63a1ba2c.exe

**Comment by huffman** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3017/HighFidelity-Beta-PR11281-7a68788c8c9b8646c3fe72f1efa84ac950b693ae.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3023/HighFidelity-Beta-PR11281-c67fcc13844c9b28e8eb2b304e87aa5f3033e2ae.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3028/HighFidelity-Beta-PR11281-eb2e87a7982897f1612257e5e111bf9f49e3fe0e.exe

**Pull request merged**